.envrc

```diff
@@ -65,7 +70,9 @@ ${red}${bold}direnv FATAL: ${@}
 ${reset}
 EOF
     # When reporting to Sentry, this will allow grouping the errors differently
+    # NOTE: The first line of the output is used to group issues
     error_message=( "${@}" )
+    log_level="error"
```

Comment:
This will make issues that are reported via `die` report red issues instead of yellow ones. We can adjust reporting in these cases.